lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -207,24 +206,16 @@ export default function SubmitGradeForm({
 
   return (
     <>
-      <form autoComplete="off">
+      <form autoComplete="off" onSubmit={onSubmitGrade} ref={formRef}>
         <label htmlFor={gradeId} className="font-semibold text-xs">
           Grade (Out of {scoreMaximum})
         </label>
         <div className="flex">
           <span className="relative w-14">
-            {validationMessage && (
-              <ValidationMessage
-                message={validationMessage}
-                open={showValidationError}
-                onClose={() => {
-                  // Sync up the state when the ValidationMessage is closed
-                  setValidationError(false);
-                }}
-              />
-            )}
             <Input
               classes={classnames(
+                // Hide up/down arrow buttons
+                '[appearance:textfield] [&::-webkit-outer-spin-button]:appearance-none [&::-webkit-inner-spin-button]:appearance-none',
```

Comment:
I got this from stackoverflow https://stackoverflow.com/a/75872055

I tested it in Firefox and Chrome and seems to do the trick. If we want to reuse it, we might want to create some kind of tailwind plugin.

We could event pre-set this in frontend-shared if we want all number input to look like regular text inputs, or provide a prop that sets these classes conditionally.